### PR TITLE
[LLM Runtime] Remove use_cache in WOQ

### PR DESCRIPTION
**Pull request by a32543254**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

Past:
| use_cache | Bool | Use local quantized model if file exists (default: False) |

Now:
We auto detect the file, don't need consumers to aware this args.

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by airMeng** [comment]
can you give a brief demonstration that what ```use_cache``` is, why it was added and now why it is deperated?

**Comment by a32543254** [comment]
> | use_cache | Bool | Use local quantized model if file exists (default: False) |
> 
> > > > > > > > >
> 
> Now, we auto detect the file, don't need consumers to aware this args.

**Comment by airMeng** [comment]
> Now, we auto detect the file, don't need consumers to aware this args.

Can you link the pr since which models can auto detected?

**Review by airMeng — APPROVED** [review]

**Comment by a32543254** [comment]
> > Now, we auto detect the file, don't need consumers to aware this args.
> 
> Can you link the pr since which models can auto detected?

all the models can auto detected, as long as you already get the converted model which means *.bin format.

File: intel_extension_for_transformers/llm/runtime/graph/__init__.py [hunk 1]
```diff
@@ -106,15 +106,18 @@ def init(self, model_name, use_quant=True, use_cache=False, use_gptq=False, **qu
             self.bin_file = fp32_bin
         else:
             self.bin_file = quant_bin
-        if use_cache and os.path.exists(self.bin_file):
+
+        if os.path.exists(self.bin_file):
+            print("{} existed, will use cahce file. Otherwise please remove the file".
```
Comment (hshen14) [inline intel_extension_for_transformers/llm/runtime/graph/__init__.py:111]: cache

Comment (a32543254) [reply intel_extension_for_transformers/llm/runtime/graph/__init__.py]: Thanks！
fixed

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**